packages/libs/react-ui/src/components/NavHeader/NavHeader.css.ts

```diff
@@ -0,0 +1,132 @@
+import { sprinkles, vars } from '../../styles';
+
+import { style } from '@vanilla-extract/css';
+
+export const containerClass = style([
+  sprinkles({
+    alignItems: 'stretch',
+    backgroundColor: '$gray90',
+    color: '$gray40',
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    height: '$16',
+    justifyContent: 'flex-start',
+    overflow: 'hidden',
+    position: 'relative',
+  }),
+  {
+    selectors: {
+      '&:hover': {
+        textDecoration: 'none',
+      },
+      '&:active': {
+        color: vars.colors.$negativeContrast,
+      },
+    },
+  },
+]);
+
+export const logoClass = style([
+  sprinkles({
+    display: 'flex',
+    marginLeft: '$3',
+  }),
+  {
+    alignSelf: 'center',
+    zIndex: 1,
+  },
+]);
+
+export const navWrapperClass = style([
+  sprinkles({
+    alignItems: 'stretch',
+    display: 'flex',
+    justifyContent: 'center',
+  }),
+]);
+
+export const navListClass = style([
+  sprinkles({
+    alignItems: 'stretch',
+    display: 'flex',
+    justifyContent: 'center',
+  }),
+  {
+    alignSelf: 'center',
```

Comment:
can you apply this to via the parent to avoid adding these extra styles?

Reply:
To the container? That wouldn't work.